config/test.exs

```diff
@@ -36,3 +36,9 @@ config :cadet, Cadet.Auth.Guardian,
   secret_key: "4ZxeVrSvCJlmndrFL7tBpnZsTc/rOQygVIyscAMY1oKKzkKi7hkjXl9F1f28Jap8"
 
 config :arc, definition: Arc.Storage.Local
+
+config :cadet,
```

Comment:
Might be better to add this to `config.exs`? Or probably include this as well inside `prod.secrets.exs` as well.

Reply:
hmm, they'll be two different paths... will that make sense?

Reply:
Well, then the one here would be just for tests, and the one in `prod.secrets.exs` is for production only.

Reply:
Bump. I think just provide this in 3 separate locations for `test.exs`, `dev.exs`, and `prod.secrets.exs.example` to be provided for production deployment